shared/middleware/helpers/storage_apps.rb

```diff
@@ -204,6 +214,27 @@ def reset_abuse(channel_id)
     0
   end
 
+  #
+  # Disables or enables automated content moderation for this project by
+  # altering the value for skip_content_moderation.
+  # @param [String] channel_id - an encrypted channel id
+  # @param [Boolean] should_skip, whether the content moderation should be
+  # skipped or not for this project.
+  # @raise [NotFound] if the channel does not exist or already has the desired
+  # value for skip_content_moderation
+  #
+  def change_content_moderation(channel_id, should_skip)
```

Comment:
Naming nit: `set` instead of `change`.  Since this is a boolean value it might be a non-issue, but with objects it matters.  "Change" implies the subject may inherit some of its past state, while "set" suggests the value after the operation will be _only_ the new information you passed it.  It's like the difference between `PUT` and `PATCH` in HTTP.